source/sync_order.rst

```diff
@@ -0,0 +1,456 @@
+403                 Forbidden – You do not have permission for this request
+405                 HTTP method not allowed
+500, 502, 503, 504  Something went wrong on Omni's end
+=================== ==============================
+
+
+Here are examples of API responses:
+
+
+If call to order status API is successful (JSON):
+
+.. code-block:: bash
+
+    HTTP/1.0 200 OK
+    Vary: Accept
+    Content-Type: application/json
+    Allow: POST, OPTIONS
+    [Redacted Header]
+
+    {
+        "order": {
+            "id": 1,
+            "number": "FR9TL74P",
+            "total": 36364.0,
+            "tax": 3636.0,
+            "serviceCharge": 0.0,
+            "grandTotal": 41000.0,
+            "tableNumber": "A1",
+            "channel": 2,
+            "notes": "No lettuce",
+            "status": 1,
+            "deliveryStatus": 10,
+            "statusText": "New",
+            "storeName": "BURGER GOD",
+            "userID": 1,
+            "created": 1564045835,
+            "paymentMethod": 1,
+            "paymentStatus": 1,
+            "user": {
+                "id": 1,
+                "name": "test",
+                "phone": "+628111111111"
+            },
+            "items": [
+                {
+                    "id": 1,
+                    "notes": "",
+                    "subtotal": 36364.0,
+                    "quantity": 1,
+                    "variant": {
+                        "id": 1,
+                        "code": "BURGER01",
+                        "sku": "BURGER01",
+                        "name": "Burger",
+                        "displayName": "",
+                        "isActive": true,
+                        "upsizedVersion": null
+                    },
+                    "modifiers": []
+                }
+            ],
+            "delivery_info": null
+        }
+    }
+
+When some fields don't validate (JSON):
+
+.. code-block:: bash
+
+    HTTP/1.0 400 BAD REQUEST
+    Vary: Accept
+    Content-Type: application/json
+    Allow: POST, OPTIONS
+    [Redacted Header]
+
+    {
+        "error_message": "Your request cannot be completed",
+        "error_code": "invalid_request"
+    }
+
+If missing or wrong authentication token:
+
+.. code-block:: bash
+
+    HTTP/1.0 401 UNAUTHORIZED
+    Vary: Accept
+    Content-Type: application/json
+    Allow: POST, OPTIONS
+    [Redacted Header]
+    
+    {"detail": "Invalid token"}
+
+If HTTP is used instead of HTTPS:
+
+.. code-block:: bash
+
+    HTTP/1.0 403 FORBIDDEN
+    Vary: Accept
+    Content-Type: application/json
+    Allow: POST, OPTIONS
+    [Redacted Header]
+
+    {"detail": "Please use https instead of http"}
+    
+    
+2. Get Order
+====================
+| URL endpoint: https://orders.upnormal.co.id/api/store/orders/get
+| Allowed method: GET
+| Requires authentication: Yes
+
+A. Request
+----------
+
+You can get a specific order's details by calling the API with these parameters
```

Comment:
For get order, it will return only the latest 15 orders.

Reply:
done